CacheManager.StackExchange.Redis.SelfCleaning.Examples/Scenarios/SelfCleaningHermeticityScenario.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Net.Http.Headers;
+using System.Text;
+using CacheManager.Core;
+using CacheManager.Core.Internal;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Examples.Scenarios
+{
+    public class SelfCleaningHermeticityScenario : BaseSingleScenario<double>
+    {
+        private ICacheManager<double>[] _cacheManagers;
+        private long[] _cacheManagersAdditionCounter;
+
+        private List<CacheItem<double>> _expectedRemovedCacheItems;
```

Comment:
Right, I meant to upgrade the scenario so it will support verification for not only the number of elements, but to their content and order. Do you think It should be seperate scenario (and therefore this list should be erased here)?

Reply:
Maybe, not necessarily, but keep in mind that for large amount of item this could make the scenario rather heavy

Reply:
Ok, so maybe I will do it in a smaller (small amount of messages) scenario... For now, removed it